closure/compiler/test/BUILD

```diff
@@ -43,7 +43,7 @@ file_test(
                "\"version\":3,\n" +
                "\"file\":\"/closure/compiler/test/hello_bin.js\",\n" +
                "\"lineCount\":1,\n" +
-               "\"mappings\":\"AAgBAA,OAAAC,IAAA,CAAgB,aAAhB;\",\n" +
```

Comment:
That's really good that the compiler is smart enough to take the wrapper into consideration when generating the sourcemap.